charts/golang/values.yaml

```diff
@@ -165,9 +165,11 @@ vault:
   enabled: false
   secrets: []
     # - name: env
-    #   path: "secret/data/env"
+    #   path: "devops/go-demo/prod/secret/data/env"
     #   value: |-
-    #     export NEWRELIC_LICENSE="{{ .Data.data.NEWRELIC_LICENSE }}"
+    #     {{- range $k, $v := .Data.data -}}
+    #        {{ $k }}={{ $v }}
+    #     {{ end -}}
```

Comment:
Should we just default to this if `value` is not supplied? That way we use this as a default, but allow them to override their template contents if they desire?

Reply:
I like the idea including the template in the values.yaml file template and (hopefully) give someone new or unfamiliar with it the clue that you can template the content of a secret however they'd like, but I'm fine with either approach.

Reply:
@GrantSheehan You can set a default to "-", which docs to explain what is going on. For example:

```
    # The below example will use a standardized template that injects a file formatted in a way that
    # allows libraries like Viper, or dotenv, to load the file at runtime.
    # - name: env
    #   path: "devops/go-demo/prod/secret/data/env"

    # The below example shows you how to build your own template value for the injected Vault file.
    #
    # - name: env
    #   path: "devops/go-demo/prod/secret/data/env"
    #   value: |-
    #     {{- range $k, $v := .Data.data -}}
    #        {{ $k }}={{ $v }}
    #     {{ end -}}
```

Reply:
I added a default for the template if it's omitted, what do you think?